dev-libs/soci/soci-4.0.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Simple Open (Database) Call Interface - write SQL in C++"
+HOMEPAGE="https://github.com/SOCI/soci"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
We generally dislike having `${PN}` in here, as it's not a changeable variable, and makes copy-pasting/grepping harder. Also imagine Github had case-sensitive URL matching.